momepy/elements.py

```diff
@@ -201,7 +201,8 @@ def enclosed_tessellation(
         The maximum distance between points after discretization. A right value is
         a sweet spot between computational inefficiency (when the value is too low)
         and suboptimal resulting geometry (when the value is too large). The default
-        is empirically derived for the use case on
+        is empirically derived for the use case on building footprints represented in
+        meters. By default 0.5
```

Comment:
Not necessarily. Could be feet or something. It is represented in units of the CRS.

Reply:
Sure, I was copying from the docstring on line 80, which was essentially the same. I can update both to map units.